undead-discord.css

```diff
@@ -33,6 +33,64 @@
   border-bottom: 1px solid var(--darkest-bg-color);
 }
 
+/* The following 3 blocks color the keybind-shortcut modal and buttons */
+.ddr-arrows { display: none; }
+
+.keybind-shortcut > span {
+    color: #4F545C !important;
+    background-color: var(--secondary-bg-color) !important;
+    border: none;
+    box-shadow: none;
+    height: 21px;
+}
+
+.keybind-shortcut > span:active {
+    color: #4F545C !important;
+    border: none;
+    box-shadow: none;
+    height: 21px;
+}
+
+/* Colors the help modal (F1 key) */
+.inner-1_1f7b > .modal-content,
+.inner-1_1f7b > .modal-content > .header,
+.inner-1_1f7b > .modal-content > .footer {
+    background-color: var(--bg-color) !important;
+}
+
+.inner-1_1f7b > .modal-content > .form-inner {
+    background-color: var(--secondary-bg-color) !important;
+}
+
+/* Replaces whitish border color with the dark red color */
+.inner-1_1f7b > .modal-content > .footer {
+    // border-top: 1px solid var(--highlight-color);
```

Comment:
`//` isn't even a valid CSS comment. Please just remove this instead of commenting it out. Same for all the other places where you used this.

Reply:
Oh boy, thought I removed all my commented out shit xD